src/components/assets/css/Authenticate.css

```diff
@@ -0,0 +1,64 @@
+h2 { 
```

Comment:
No, need for this CSS file when you will use styled components.

Reply:
bro it is needed for icons and some of the styles, please check it.

Reply:
Refer this - https://github.com/anushbhatia/Social-Scheduler/blob/backend/src/components/About.jsx